rir/src/compiler/translations/rir_2_pir/rir_2_pir_compiler.cpp

```diff
@@ -64,28 +63,47 @@ void Rir2PirCompiler::compileClosure(rir::Function* srcFunction,
         srcFunction, formals.names, closureEnv, [&](Closure* pirFunction) {
             Builder builder(pirFunction, closureEnv);
             Rir2Pir rir2pir(*this, srcFunction);
+            if (isVerbose()) {
+                std::cout << "\n\n*********************************************"
```

Comment:
Nitpick: I know this is because `clang-format` want it this way, but I actually find this to be worse than just going past the 80 column max. Is there a way to tell `clang-format` to ignore some lines?